src/plugins/presentation_panel/public/panel_component/_presentation_panel.scss

```diff
@@ -184,3 +169,60 @@
   padding-left: $euiSizeS;
   z-index: $euiZLevel1;
 }
+
+.embPanel__hoverActionsAnchor {
+  position: relative;
+  height: 100%;
+
+  .embPanel__hoverActionsWrapper {
+    height: $euiSizeXL;
+    position: absolute;
+    top: 0;
+    display: flex;
+    justify-content: space-between;
+    padding: 0 $euiSizeL;
+    flex-wrap: nowrap;
+    min-width: 100%;
+    z-index: -1;
+    pointer-events: none; // Prevent hover actions wrapper from blocking interactions with other panels
+  }
+
+  .embPanel--dragHandle {
+    .embPanel--dragHandle:hover {
+      cursor: move;
+    }
+
+    img {
+      pointer-events: all !important;
+    }
+  }
```

Comment:
```suggestion
  .embPanel--dragHandle {
    cursor: move;
      
    img {
      pointer-events: all !important;
    }
  }
```

Fix cursor on drag handle.

Reply:
Applied in https://github.com/elastic/kibana/pull/182535/commits/26033584e6a47496f3c193842f8cbc8de1bb6bd0